packages/reactivue/src/component.ts

```diff
@@ -4,13 +4,26 @@ import { isDev } from './env'
 import { invokeLifeCycle } from './lifecycle'
 import { InternalInstanceState, LifecycleHooks } from './types'
 
-let _id = 0
-const _vueState: Record<number, InternalInstanceState> = {}
+type ReactiveEffects = Record<number, InternalInstanceState>
+
+declare global {
+  interface Window { _reactivueState: ReactiveEffects }
+}
+
+const _vueState: ReactiveEffects = window?._reactivueState || {}
```

Comment:
Ehmm, I don't think it's good to expose it on window, maybe we can found out a better solution

Reply:
I retrict it to dev only now.